guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -93,7 +93,7 @@ Complete the following steps on the connected {ProjectServer}.
 [options="nowrap" subs="attributes"]
 ----
 [{RepoRHEL7ServerAnsible}]
-name=Ansible {SatelliteAnsibleVersion} RPMs for Red Hat Enterprise Linux 7 Server x86_64
+name=Ansible {SatelliteAnsibleVersion} RPMs for {RHEL} 7 Server x86_64
```

Comment:
I'm not sure this is correct. Previously it used `Red Hat` and now `Red&nbsp;Hat`. I'm not sure if the non breaking space can cause troubles.

Reply:
Reverting it back to Red Hat Enterprise Linux